### PR TITLE
Update DG for Edit command

### PR DESCRIPTION
**Pull request by manu2002g**

### CODE REVIEW

**Comment by viki0526** [comment]
LGTM! Sequence and Class diagrams can be added next time

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -200,6 +200,18 @@ allow subclasses of `Tag` to be tagged to a person. Currently, there are 4 of su
...
+### Original implementation
+The edit command uses an `EditPersonDescriptor` to store the new information that is to be changed in the person. The
+`EditCommandParser` parses the information input and then creates an `EditPersonDescriptor` where the unchanged fields
+are copied over from the existing person and the fields to be overwritten are changed. The Find command then takes in
+the descriptor and simply changes the persons attribute values to the values stated in the descriptor.
+
+### Current Implementation
+The edit command has now been upgraded to support the functionality for overwriting multiple tag lists.
+
+
```
Comment (fredtwt) [inline docs/DeveloperGuide.md:214]: Good start! Do remember to add in UML diagrams in future! 👍

**Review by viki0526 — COMMENTED** [review]
LGTM

**Review by ongkimlai — APPROVED** [review]
LGTM! Agree with Vikrant, could add in some UML diagrams

**Review by fredtwt — APPROVED** [review]
LGTM!

**Pull request merged**